tests/unit/name-request-mixin.spec.ts

```diff
@@ -211,10 +208,10 @@ describe('Name Request Mixin', () => {
     expect(vm.isNrInvalid(nr)).toBe(true)
 
     nr.request_action_cd = 'CNV'
-    expect(vm.isNrInvalid(nr)).toBe(true)
+    expect(vm.isNrInvalid(nr)).toBe(false) // valid
 
     nr.request_action_cd = 'CHG'
-    expect(vm.isNrInvalid(nr)).toBe(true)
+    expect(vm.isNrInvalid(nr)).toBe(false) // valid
```

Comment:
The Name Request Mixin now allows 3 types of NR (CNV, CHG, REH). It's the component itself that will check the NR type against the allowed types from the resources (depending on the filing type).